Pipfile.lock

```diff
@@ -1033,11 +1044,11 @@
         },
         "moto": {
             "hashes": [
-                "sha256:8bb8e267d9b948509d4739d81d995615a193d2c459f5c0a979aaeb0d3bd4b381",
-                "sha256:cbe8ad8a949f519771e5d25b670738604757fb67cd474d75d14c20677582e81f"
+                "sha256:6fb81f500c49f46f19f44b1db1c2ea56f19f90d0ca6b944866ae0f0eeab76398",
+                "sha256:a9529f295ac786ea80cdce682d57170f801c3618c3b540ced29d0473518f534d"
             ],
             "index": "pypi",
-            "version": "==3.1.16"
+            "version": "==4.0.1"
```

Comment:
moto and a few other packages incidentally got major version upgrades. pointing it out in case there may be breaking changes we need to worry about that won't be caught in tests.